indexFormat/specification.md

```diff
@@ -807,7 +807,13 @@ export interface Project extends V {
 	/**
 	 * Optional the content of the project file, `base64` encoded.
 	 */
-	contents?: string;
+  contents?: string;
+
+  /**
+   * An optional hash of the files included in the project, `base64` encoded. projectHash should be the same

```

Comment:
No need to define the encoding or any specifications for this string - opaque string that language determines. Specify size - 512 chars maybe?